recipe/meta.yaml

```
@@ -34,13 +34,19 @@ requirements:
 # We can`t enable pytest because conftest uses the coincidence package, which is unavailable on the main channel.
 # The policy states that packages are not specifically built for testing.
 test:
+  source_files:
+    - tests
   imports:
     - consolekit
+  requires:
+    - pip
+    - pytest >=6.0.0
+    - coincidence >=0.1.0
+    - pytest-regressions >=2.0.2
```

https://github.com/domdfcoding/consolekit/blob/v1.8.1/tests/requirements.txt#L18
```suggestion
    - pytest-regressions >=2.0.1
```

---

Where does this version number come from?

---

link ⏫